src/user/contributing.rst

```diff
@@ -40,9 +40,9 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
    `conda-forge.github.io repository
    <https://github.com/conda-forge/conda-forge.github.io>`_.
 2. `Clone <https://help.github.com/articles/cloning-a-repository/>`_ this fork onto your computer.
-3. `Check out
+3. `Create a new branch
    <https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging>`_
-   a new branch deriving from ``master`` to do your work.
+   to make all the changes.
```

Comment:
It would be good to leave some text about the master branch in there, even if the link goes over that. It's a standard thing to do to make a branch from master/main, but might not be obvious to everyone, and we want to be as novice friendly as we can!